mainMARS.s

```diff
@@ -217,12 +217,64 @@ max:
   pop_ra_and_return
 
 # Power operation.
+.data 
+power_a_message: .asciiz "Enter the a in (a ^ b):\n"
+power_b_message: .asciiz "Enter the b in (a ^ b):\n"
+zero_exponent_message: .asciiz "The result equals 1 "
 
 .text
-power:
-  push_ra
```

Comment:
`push_ra` is necessary here. Don't remove it.

Reply:
sorry I don't mean to remove it